core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -367,7 +367,7 @@ object ConfigCommand extends Config {
       println(s"Completed updating default config for $entityType in the cluster.")
   }
 
-  private def describeConfig(adminClient: Admin, opts: ConfigCommandOptions): Unit = {
+  private[admin] def describeConfig(adminClient: Admin, opts: ConfigCommandOptions): Unit = {
```

Comment:
Is this visibility just for testing?

Reply:
yes, the alterConfigs method above this had the same visibility too for the same reason.